RosBE-Unix/Base-i386/scripts/RosBE-rc

```diff
@@ -1,3 +1,9 @@
+# extend user's RC file
+source $HOME/.bashrc
```

Comment:
Please check first if that file exists (using `if [[ -f $HOME/.bashrc ]]`). It may not for people primarily using other shells.
I also hope this move doesn't pull in wild environment variables that screw us later...

Reply:
> I also hope this move doesn't pull in wild environment variables that screw us later
That is something I have in my local installation for several month, so I decided - maybe other people want that too :)

Regarding regression paths, I see one in user's `.bashrc` may have some additions stuff in `$PATH`